debezium-core/src/main/java/io/debezium/data/VariableScaleDecimal.java

```diff
@@ -22,26 +21,24 @@
 public class VariableScaleDecimal {
     public static final String LOGICAL_NAME = "io.debezium.data.VariableScaleDecimal";
     public static final String SCALE_FIELD = "scale";
-    public static final String VALUE_FIELD = "value";
-
+    public static final Struct ZERO = fromLogical(schema(), DebeziumDecimal.ZERO);
     /**
      * Returns a {@link SchemaBuilder} for a VariableScaleDecimal. You can use the resulting SchemaBuilder
      * to set additional schema settings such as required/optional, default value, and documentation.
-     * 
+     *
      * @return the schema builder
      */
     public static SchemaBuilder builder() {
-        return SchemaBuilder.struct()
+        return DebeziumDecimal.builder()
                 .name(LOGICAL_NAME)
                 .version(1)
                 .doc("Variable scaled decimal")
-                .field(SCALE_FIELD, Schema.INT32_SCHEMA)
-                .field(VALUE_FIELD, Schema.BYTES_SCHEMA);
+                .field(SCALE_FIELD, Schema.OPTIONAL_INT32_SCHEMA);
```

Comment:
That's not a compatible change in terms of the schema evolution rules. But I'm wondering why it's done at all? No change should be needed here when going the String route exclusively. Or did you try to add special value support to `VariableScaleDecimal`, too?

Reply:
Thanks for spotting, it is a leftover from the situation where it was possible to store either value or special value.